internal/server/resource/resource.go

```diff
@@ -103,13 +121,28 @@ func (index *SearchIndex) Update(
 			}
 		}
 	}
-	approxNumPv := len(strings.Split(nodeID, "_"))
-	if approxNumPv == 1 {
-		// when approxNumPv is 1, most likely a non human curated PV
-		approxNumPv = nonHumanCuratedNumPv
+	numPV := 0
+	if len(svDefinition) > 0 {
+		svDefParts := strings.Split(svDefinition, ",")
+		for _, defPart := range svDefParts {
+			// don't include required properties when counting PVs
+			if len(defPart) >= requiredPVDefPrefixLength {
+				prefix := defPart[0:requiredPVDefPrefixLength]
+				if _, ok := requiredPVDefPrefixes[prefix]; ok {
+					continue
+				}
+			}
```

Comment:
Among the cases where num-pvs match, we should pick the one with fewer known props.

For example, pt=Person,mp=count,st=measuredValue should come before pt=Person,mp=count,st=measuredValue,md=Area

Reply:
done